pgb_utils/requirements.txt

```diff
@@ -1,14 +1,7 @@
 aplpy
-astropy>=4.0  # allow create Table from row dicts with different keys
-# astropy==3.2.1
-# astropy-healpix==0.6
-# beautifulsoup4==4.8
+astropy==3.2.1  # required by figures.plot_stamp(), see comments there
```

Comment:
I don't think this comment is necessary. Specifically because I don't know at a glance where  `there` is and pointing at the code is just another thing to maintain. I think it's assumed by using `==`` that using other versions might break things.

Reply:
I specifically asked @troyraen to document why versions were pinned.  I appreciate the point that `figures.plot_stamp()` doesn't uniquely identify the code, but it does give someone a pointer of what to check for.  Someday we will need to update to AstroPy 4.0 someday for some other function and we need to write down somewhere what to check for breakage when we update.

I think if you say `>=` then you don't really have to say why.  But if you pin a version, then you're specifically saying don't upgrade beyond that because you'll break something.  But you know that that package is going out of date at some point and you know someone will have to updated why.  Pointers, even if it's just a hint, are really key in making sure you can check that you're not going to break the thing that led to pinning the version in the first place.